stat.c

```diff
@@ -472,9 +472,12 @@ static void show_ddir_status(struct group_run_stats *rs, struct thread_stat *ts,
 		display_lat("clat", min, max, mean, dev, out);
 	if (calc_lat(&ts->lat_stat[ddir], &min, &max, &mean, &dev))
 		display_lat(" lat", min, max, mean, dev, out);
+	if (calc_lat(&ts->clat_prio_stat, &min, &max, &mean, &dev) && ddir == DDIR_READ)
+		display_lat("PRIO_clat", min, max, mean, dev, out);
```

Comment:
I don't think the other fields use mixed word case like this. Also maybe swap the order of comparisons, i.e. check `DDIR_READ` first?

Reply:
Good idea, and I'll change the case.